samples/Zirku/Zirku.Client2/wwwroot/index.js

```diff
@@ -0,0 +1,29 @@
+﻿oidc.Log.setLogger(console);
+
+const userManager = new oidc.UserManager({
+    authority: 'https://localhost:44319',
+    scope: 'openid offline_access api1',
+    client_id: 'spa',
+    redirect_uri: window.location.origin + '/signin-callback.html',
+    silent_redirect_uri: window.location.origin + '/signin-silent-callback.html',
```

Comment:
Note: the silent authentication mechanism works by sending `prompt=none` authorization requests in invisible iframes: with the ban of third-party cookies initiated by many browser vendors, this approach will no longer work reliably as soon as you host the SPA and the authorization server on different domains/origins.

`userManager.signinSilent()` should be smart enough to use the refresh token when one was retrieved, so we should be able to remove `silent_redirect_uri` and the corresponding .html file.

Reply:
Done. Though it's unclear to me what's the difference between doing the request from an iframe or directly from the spa. AFAIK they share the same domain. 🤔

Reply:
The issue is when the SPA and the identity servers are on different domains: browsers started preventing iframes created by domain1.com from sending cookies belonging to domain2.com (no cookie is received), which kills features like silent sign-in.